docs/_ug/Layout.md

```diff
@@ -0,0 +1,18 @@
+<!-- markdownlint-disable-file first-line-h1 -->
+
+### Layout
```

Comment:
Is this file included anywhere? I noticed that you deleted this section from `Installation.md` but did not add the include statement. Apologies if I missed it somewhere else.